src/main/resources/templates/tableHeader.vm

```diff
@@ -6,7 +9,14 @@
     <th></th>
 </tr>
 <tr>
-    <th id="stats-header-feature">Feature</th>
+    #if($tagPage)
+        <th id="stats-header-tag">Tag</th>
```

Comment:
some day it will become big if-else

Instead I suggest to use already defined value https://github.com/damianszczepanik/cucumber-reporting/blob/master/src/main/resources/templates/pages/tagOverview.vm#L184 and in that case changes in java are not required